README.md

```diff
@@ -40,6 +40,12 @@ black .
 pylint prommis
 ```
 
+#### Running spell checker (Typos)
+
+```sh
+typos
+```
+
```

Comment:
```suggestion

Note: if the `typos` executable is not found, it can be installed by running `conda install --yes -c conda-forge typos` after activating the `prommis-dev` Conda environment.

```